src/header/header.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './header.css';
```

Comment:
Why dont you use css modules?

Reply:
When i run eject command i got only two webpack config files in my config folder , As per the max and others instructions have to changes for modules in webpack.config.dev.js and webpack.config.prod.js but i got only one js file (webpack.config.js) so i didn't  change modules and use.